packages/cashier/src/pages/account-transfer/account-transfer-form/account-transfer-form.tsx

```diff
@@ -146,7 +146,7 @@ const AccountTransferForm = observer(
             });
             if (!is_ok) return message;
 
-            if (selected_from.balance && Number(selected_from.balance) < Number(amount))
+            if (typeof selected_from.balance !== 'undefined' && Number(selected_from.balance) < Number(amount))
```

Comment:
need to check if  selected.balance is not an undefined and then can do the comparasion
cannot use ?? `if (selected_from.balance ?? Number(selected_from.balance) < Number(amount))` because we will jump into if condition since selected_from.balance is a truthy value